build.sbt

```diff
@@ -45,7 +46,7 @@ lazy val root = (project in file("."))
       "com.disneystreaming" %% "weaver-scalacheck" % Versions.Weaver,
       "org.scalatest" %% "scalatest" % "3.2.16", // just for `shouldNot compile`
       "com.dimafeng" %% "testcontainers-scala-localstack-v2" % "0.40.17",
-      "com.amazonaws" % "aws-java-sdk-core" % "1.12.523" exclude ("*", "*"), // fixme after release of https://github.com/testcontainers/testcontainers-java/pull/5827
+      "com.amazonaws" % "aws-java-sdk-core" % "1.12.526" exclude ("*", "*"), // fixme after release of https://github.com/testcontainers/testcontainers-java/pull/5827
```

Comment:
Since we bump it, can we drop the exclusion rule? https://github.com/testcontainers/testcontainers-java/pull/5827 seems merged

Reply:
oh since i last time checked it was merged, but not available in testcontainers-scala. now it is thanks